hack/lint.sh

```
@@ -2,14 +2,17 @@
 if [ "$#" -ne 1 ]; then
     echo "Expected command line argument - container runtime (docker/podman) got $# arguments: $@"
     exit 1
 fi
 
-$1 run --security-opt label=disable --rm -v $(pwd):/app -w /app -e GO111MODULE=on golangci/golangci-lint:${VERSION} \
-	golangci-lint run --verbose --print-resources-usage \
-	--modules-download-mode=vendor --timeout=15m0s && \
+USERNS_FLAG=""
+[[ "$1" == "podman" ]] && USERNS_FLAG="--userns=keep-id"
+
+$1 run --security-opt label=disable --rm $USERNS_FLAG -v $(pwd):/app -w /app -e GO111MODULE=on docker.io/library/golang:1.26.0 \
```

Shellcheck (0.11.0)

[warning] Quote this to prevent word splitting.

(SC2046)